client/store.js

```diff
@@ -244,6 +254,12 @@ store.setupKresus = function(cb) {
         has(world, 'accounts');
         has(world, 'operations');
 
+        let defaultAccountId = store.getDefaultAccountId();
+        if (defaultAccountId && !data.currentAccountId)
+        {
+            data.currentAccountId = defaultAccountId;
```

Comment:
That would only work if defaultAccountId is defined. Which is not the case until the user stars a bank account for the first time.